test/OpenTelemetry.AutoInstrumentation.Tests/OpenTelemetry.AutoInstrumentation.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" />
     <PackageReference Include="Xunit.SkippableFact" />
+    <PackageReference Include="log4net" />
```

Comment:
I would consider moving tests related to the dependencies to the separate package. To avoid brining here transitive dependencies.